website/addons/osfstorage/routes.py

```diff
@@ -112,6 +112,24 @@
             views.osfstorage_copy_hook,
             json_renderer,
         ),
+
+        Rule(
+            [
+                '/project/<pid>/osfstorage/<fid>/tags/<tag>/',
```

Comment:
@brianjgeiger No, `<tag>` does not necessarily need to be in the path for v1. In fact, I just checked that project tags now use the JSON request body. The legacy paths still exist but are unused AFAIK.

```python
        # Add / remove tags
        Rule([
            '/project/<pid>/tags/',
            '/project/<pid>/node/<nid>/tags/',
            '/project/<pid>/tags/<tag>/',
            '/project/<pid>/node/<nid>/tags/<tag>/',
        ], 'post', project_views.tag.project_add_tag, json_renderer),
```

https://github.com/CenterForOpenScience/osf.io/blob/5b95f83e57f61abd20e045c61adaa71b4dd6dcd5/website/routes.py#L1219-L1225

@hmoco So for now, let's go with `/project/<pid>/osfstorage/<fid>/tags/` for this route, using the same JSON body request format as project tags.